scripts/jobExecCondor.sh

```diff
@@ -53,19 +57,23 @@ stageOut() {
 	# try to copy n times, increasing wait each time
 	TMPWAIT=0
 	for ((i=0; i < $NUMREP; i++)); do
-		xrdcp $XRDARGS $INPUT $OUTPUT
+		if [ $GFAL -eq 1 ]; then
+			gfal-copy $XRDARGS $INPUT $OUTPUT
+		else
+			xrdcp $XRDARGS $INPUT $OUTPUT
+		fi
 		XRDEXIT=$?
 		if [ $XRDEXIT -eq 0 ]; then
 			return 0
 		fi
 		# in case of bad exit, wait and try again
 		TMPWAIT=$(($TMPWAIT + $WAIT))
-		if [ $QUIET -eq 0 ]; then echo "Exit code $XRDEXIT, failure in xrdcp. Retry after $TMPWAIT seconds..."; fi
+		if [ $QUIET -eq 0 ]; then echo "Exit code $XRDEXIT, failure in xrdcp/gfal-copy. Retry after $TMPWAIT seconds..."; fi
```

Comment:
can you make the warning message specific (either xrdcp or gfal-copy, not both)? (also below)

Reply:
This should be fixed now.